src/utils/netlifyUtils.ts

```diff
@@ -0,0 +1,5 @@
+import axios from "axios";
+
+export async function deploySite(): Promise<void> {
+  await axios.post(`${process.env.P4NTH3RBLOG_BUILD_HOOK}`);
```

Comment:
This could be 
```js
await axios.post(process.env.P4NTH3RBLOG_BUILD_HOOK);
```
I don't think there is any need for interpolation and would give nanoseconds of performance improvement

Reply:
Linting moaned at me :D

Reply:
Fair enough! Linkting is the biggest pain of dev! :P